code/cgame/cg_draw.c

```diff
@@ -554,17 +554,15 @@ void CG_DrawTeamBackground( int x, int y, int w, int h, float alpha, int team )
 
 	hcolor[3] = alpha;
 	if ( team == TEAM_RED ) {
-		hcolor[0] = 1;
-		hcolor[1] = 0;
-		hcolor[2] = 0;
+		hcolor[0] = 1.0f;
+		hcolor[1] = 0.0f;
+		hcolor[2] = 0.0f;
 	} else if ( team == TEAM_BLUE ) {
-		hcolor[0] = 0;
-		hcolor[1] = 0;
-		hcolor[2] = 1;
+		hcolor[0] = 0.0f;
+		hcolor[1] = 0.1f;
+		hcolor[2] = 1.0f;
 	} else {
-		hcolor[0] = 0;
-		hcolor[1] = 0;
-		hcolor[2] = 0;
+		return;
```

Comment:
this was added specifically to make UI easier to read in VR, if you want to be able to disable it, add a cvar (or ideally a menu option) to allow for this